ssd_keras/ssd_box_encode_decode_utils.py

```diff
@@ -67,6 +73,21 @@ def iou(boxes1, boxes2, coords='centroids'):
 
     return intersection / union
 
+
+if NUMBA_INSTALLED:
+    @numba.jit(nopython=True)
+    def iou_specialized(boxes1, boxes2):
+        boxes2b = np.expand_dims(boxes2, axis=0)
+
+        intersection = np.maximum(0, np.minimum(boxes1[:,1], boxes2b[:,1]) - np.maximum(boxes1[:,0], boxes2b[:,0])) * np.maximum(0, np.minimum(boxes1[:,3], boxes2b[:,3]) - np.maximum(boxes1[:,2], boxes2b[:,2]))
+        union = (boxes1[:,1] - boxes1[:,0]) * (boxes1[:,3] - boxes1[:,2]) + (boxes2b[:,1] - boxes2b[:,0]) * (boxes2b[:,3] - boxes2b[:,2]) - intersection
+
+        return intersection / union
```

Comment:
Here I didn't change anything compared to original `iou` except for removal of some unneeded checks (to allow `nopython=True`).